ludwig/schema/schema.py

```diff
@@ -1,18 +1,13 @@
-#! /usr/bin/env python
-# Copyright (c) 2020 Uber Technologies, Inc.
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+# Module structure:
```

Comment:
`ludwig/schema/schema.py` is weird. can you move this function into `ludwig/schema/__init__.py` instead?

Reply:
moved

Reply:
this actually causes a circular import error

>  I don't know enough about how python does submodule initialization but it seems as though to import schema.utils (which is used in e.g. the combiner and trainer files) it will also pull in that init file (which itself pulls in schemas from those trainer/combiner files)

> something I talked about with @justinxzhao in the past was moving the trainer, combiner, optimizer schemas into this new ludwig.schema module as well. I was going to do that in a follow up PR, but should I do this now?

Reply:
Going to move the other schemas over into this module, per conversation with @tgaddair